url_handlers_using_quarkus/src/main/java/org/acme/getting/started/ide-handler

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Delete first 22 characters (ide://open?url=file://)
```

Comment:
why not just pass the full url directly into starfish ?

Reply:
sure we can do that!!